keps/sig-api-machinery/2558-publish-version-openapi/README.md

```diff
@@ -190,7 +192,7 @@ type Frobber struct {
   Param  string `json:"param"
   // width indicates how wide the object is.
   // +optional
-  // +lifecycle:kubernetes:minVersion=v1.20,status=alpha,featureGate=Frobber2D
+  // +lifecycle:component=kubernetes,minVersion=v1.20,status=alpha,featureGate=Frobber2D
```

Comment:
Above it says built-ins are a non goal, so I don't understand what the featureGate can be a reference to?

Reply:
> Above it says built-ins are a non goal

Built-in structs are a non-goal but built-in fields are a part of the goals.

We had originally considered supporting structs as well but decided to start with fields for now - https://github.com/kubernetes/enhancements/pull/2560#issuecomment-839008443

Reply:
I don't know what it would mean to support a struct rather than a field -- all structs that appear in the api appear as a field of a parent type, no?

Non-goals need to be updated if that is not actually a non-goal :)

Reply:
> Non-goals need to be updated if that is not actually a non-goal :)

Done!